climada/entity/exposures/litpop/litpop_new_playground.py

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env python3
```

Comment:
Is this a temporary file? The name seems temporary, and all the CLIMADA notice about license are missing.

Reply:
ups forgot to delete it, thanks :)